docs/01-overview/main-areas/observability/obsv-10-useful-links.md

```diff
@@ -8,10 +8,10 @@ If you're interested in learning more about the Observability area, check out th
 
 - Check out the different ways to [access the logs](../../../04-operation-guides/operations/obsv-01-access-logs.md) provided by Kubernetes and Loki.
 
-- See how to [access and expose](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) the services Grafana<!-- , Jaeger, --> and Kiali.
+- See how to [access and expose](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) the services Grafana, Jaeger, and Kiali.
 
 - Find out what to do when the [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md) or when [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/obsv-02-troubleshoot-jaeger-shows-few-traces.md).
 
-- Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md), [logging](../../../05-technical-reference/00-architecture/obsv-02-architecture-logging.md), and [tracing](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md) components.
+- Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-monitoring.md), [logging](../../../05-technical-reference/00-architecture/obsv-02-architecture-logging.md), and [tracing](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md) components.
 
-- Look up the most common configuration parameters for [Alertmanager](../../../05-technical-reference/00-configuration-parameters/obsv-01-configpara-alertmanager.md), [Grafana](../../../05-technical-reference/00-configuration-parameters/obsv-02-configpara-grafana.md), [Prometheus](../../../05-technical-reference/00-configuration-parameters/obsv-03-configpara-prometheus.md), [Loki](../../../05-technical-reference/00-configuration-parameters/obsv-11-configpara-logging.md), and [Jaeger](../../../05-technical-reference/00-configuration-parameters/obsv-21-configpara-tracing.md).
+- Find the configuration parameters for [Monitoring](../../../05-technical-reference/00-configuration-parameters/obsv-01-configpara-monitoring.md), [Logging](../../../05-technical-reference/00-configuration-parameters/obsv-02-configpara-logging.md), [Tracing](../../../05-technical-reference/00-configuration-parameters/obsv-03-configpara-tracing.md), and [Kiali](../../../05-technical-reference/00-configuration-parameters/obsv-04-configpara-kiali.md).
```

Comment:
These documents do not exist anymore. I assume this is what you meant when you wrote in a DM that you decided to get rid of the table and just link to the charts? 

```suggestion
- Find the [configuration parameters for Monitoring, Logging, Tracing, and [Kiali](../../../05-technical-reference/00-configuration-parameters/obsv-01-configpara-observability.md).
```
Or link directly to the charts, but I think the additional information on what to do with them that's included in the linked document is also valuable, so I'd use the link to the document.

Reply:
*facepalm* yep the consolidation into 1 doc happened after I had adjusted these links. Thanks for spotting!